src/components/default-page-header.less

```diff
@@ -1,10 +1,18 @@
-.DefaultPageHeader {
+.DefaultPageHeader {}
+
+.DefaultPageHeader-container {
   .m-page-content;
-}
+  background-color: @edu-white;
+  z-index: @edu-z-index-sticky-content-layer;
 
-.DefaultPageHeader-logo {
-  width: 50px;
-  aspect-ratio: 1;
+  &.is-sticky {
+    .m-box-shadow;
```

Comment:
I added `transition: all 300ms ease;` and it made the switch a little bit smoother I feel, so maybe you wanna give it a try. It's not a huge difference though.

Reply:
That's a good idea, and it feels noticeable to me 👍 Done.

Reply:
Cool :+1: